app-menu/qvm-open-in-dvm.desktop

```diff
@@ -3,8 +3,10 @@
 ###   (typing "xdg-settings set default-web-browser firefox.desktop" will put it back to normal)
 
 [Desktop Entry]
-Name=qvm-open-in-dvm
-Exec=qvm-open-in-dvm %u
+Version=1.0
+Name=QubesOS View In DisposableVM
+Exec=/usr/bin/qvm-open-in-dvm --view-only -- %u
```

Comment:
You changed "edit in disposable" to "view in disposable", that shouldn't be the case here.